tests/cephfs/nfs/nfs_subvolume_subvolumegroup.py

```diff
@@ -0,0 +1,129 @@
+import secrets
+import string
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utilsV1 import FsUtils
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    CEPH-83574027 - Ensure creation of Subvolgroups, subvolumes works on NFS exports and run IO from nfs clients
+    Pre-requisites:
+    1. Create cephfs volume
+       creats fs volume create <vol_name>
+    2. Create nfs cluster
+       ceph nfs cluster create <nfs_name> <nfs_server>
+
+    Test operation:
+    1. Create cephfs nfs export
+       ceph nfs export create cephfs <fs_name> <nfs_name> <nfs_export_name> path=<export_path>
+    2. Crete 2 cephfs subvolume group
+    3. Create cephfs subvolume in cephfs subvolume group
+    4. Create cephfs subvolume in deafault cephfs subvolume group
+    5. Mount nfs mount with cephfs export
+       "mount -t nfs -o port=2049 <nfs_server>:<nfs_export> <nfs_mounting_dir>
+    7. Verify subvolume groups & subvolumes are created
+    6. Run IOs on both cephfs subvolumegroups & subvolumes
+
+    Clean-up:
+    1. Remove all the data in Cephfs file system
+    2. Remove all the cephfs mounts
+    3. Delete cephfs nfs export
+    """
+    try:
+        tc = "CEPH-83574027"
+        log.info(f"Running cephfs {tc} test case")
+
+        config = kw["config"]
+        build = config.get("build", config.get("rhbuild"))
+
+        fs_util = FsUtils(ceph_cluster)
+        clients = ceph_cluster.get_ceph_objects("client")
+        client1 = clients[0]
+        fs_util.prepare_clients(clients, build)
+        fs_util.auth_list(clients)
+        mon_node_ip = fs_util.get_mon_node_ips()
+        mon_node_ip = ",".join(mon_node_ip)
+        rhbuild = config.get("rhbuild")
+        nfs_servers = ceph_cluster.get_ceph_objects("nfs")
+        nfs_server = nfs_servers[0].node.hostname
+        nfs_name = "cephfs-nfs"
+        nfs_export_name = "/export1"
+        export_path = "/"
+        fs_name = "cephfs"
+        nfs_mounting_dir = "/mnt/nfs_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        if "5.0" in rhbuild:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {fs_name} {nfs_name} "
+                f"{nfs_export_name} path={export_path}",
+            )
+        else:
+            client1.exec_command(
+                sudo=True,
+                cmd=f"ceph nfs export create cephfs {nfs_name} "
+                f"{nfs_export_name} {fs_name} path={export_path}",
+            )
+        commands = [
+            f"ceph fs subvolumegroup create {fs_name} subvolume_group1",
```

Comment:
can we use create_subvolumegroup() function instead of cmd.
This will help us in code maintenance .
If there is any change to the creation command in the future, we can change it in the core function which gets reflected in all the scripts.
Same for subvolume as well